database/README.md

```diff
@@ -0,0 +1,14 @@
+This document explains the database that the orderbook api uses.
+
+We need to store orders persistently even if the orderbook service goes. For this we use a PostgreSQL database. The schema can be found in `schema.sql`.
+
+In addition to the orders, we also store trade events. This allows us to keep track of filled amounts.
+
+To prevent users from spamming the database we should when receiving an order check that
+* the erc20 spending is approved
+* valid_to is not in the past and not more than <1 year> in the future
```

Comment:
Why not more than 1 year? This would prevent us from having CMM brackets (which amount to unlimited orders with infinite validity)...

Reply:
I still had in mind that one of the design decisions for v2 is that orders should have a "short" validity period. From the perspective of the database that is nice because it creates a natural limit for how long we *have to* keep track of orders. But there is no technical reason for that. We could totally allow orders valid forever.